core/code/portal_data.js

```diff
@@ -143,46 +103,3 @@ window.findPortalLatLng = function(guid) {
 })();
 
 
-// get the AP gains from a portal, based only on the brief summary data from portals, links and fields
-// not entirely accurate - but available for all portals on the screen
-window.getPortalApGain = function(guid) {
```

Comment:
> Maybe we should just mark it deprecated and remove in some future version
that's something that will never happen.

if we keep it, it's on the IITC devs to keep it up-to-date instead of the one who is using it

Reply:
Found at least one plugin that's using it.